Sources/active-win/main.swift

```diff
@@ -34,7 +34,7 @@ for window in windows {
 	let app = NSRunningApplication(processIdentifier: appPid)!
 
 	let dict: [String: Any] = [
-		"title": window[kCGWindowName as String] as! String,
+		"title": (window[kCGWindowName as String] != nil ? window[kCGWindowName as String] as! String : "" ),
```

Comment:
This can be written simpler as:

```swift
window[kCGWindowName as String] as? String ?? ""
```

Reply:
Ok. I pushed changes with simpler version.